README.md

```diff
@@ -83,18 +89,21 @@ Heroic is built with Web Technologies like: TypeScript, React, NodeJS and Electr
 - French
 - German
 - Greek
+- Japanese
```

Comment:
Japanese was translated but was missing on readme. While Romanian was here but it was not available to select.